Assets/AppCenter/Editor/CreateManifest.cs

```diff
@@ -0,0 +1,186 @@
+                .Append(" -Destination ")
+                .Append(destinationFile)
+                .ToString();
+            processName = "cmd";
+        }
+        else if (Application.platform == RuntimePlatform.OSXEditor)
+        {
+            args = stringBuilder
+                .Append("-c \"cd ")
+                .Append(sourceFile)
+                .Append(" ; zip -r ")
+                .Append("../")
+                .Append(destinationFile)
+                .Append(" * \"")
+                .ToString();
+            processName = "/bin/bash";
+        }
+        if (processName.Length > 0) {
+            ExecuteProcess(processName, args);
+        }
+    }
+
+    private static void ExecuteProcess(string processName, string args)
+    {
+        var process = new Process()
+        {
+            StartInfo = new ProcessStartInfo
+            {
+                FileName = processName,
+                Arguments = args,
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+                UseShellExecute = false,
+                CreateNoWindow = true,
+            }
+        };
+        process.Start();
+        string output = process.StandardOutput.ReadToEnd();
+        string error = process.StandardError.ReadToEnd();
+        process.WaitForExit();
+        if (output.Length > 0 || error.Length > 0)
+        {
+            UnityEngine.Debug.Log(output + error);
+        }
+    }
+
+    public static void UnzipFile(string sourceFile, string destinationFile)
+    {
+        var stringBuilder = new StringBuilder();
+        var args = "";
+        var processName = "";
+        if (Application.platform == RuntimePlatform.WindowsEditor)
+        {
+            args = stringBuilder
+                .Append("/c powershell")
+                .Append(" -File \"")
+                .Append(AppCenterSettingsContext.AppCenterPath)
+                .Append("/AppCenter/Plugins/Android/Utility/unarchive.ps1 \"")
+                .Append(" -Source ")
+                .Append(sourceFile)
+                .Append(" -Destination ")
+                .Append(destinationFile)
+                .ToString();
+            processName = "cmd";
+        }
+        else if (Application.platform == RuntimePlatform.OSXEditor)
+        {
+            args = stringBuilder
+                   .Append("-c \"unzip ")
+                   .Append(sourceFile)
+                   .Append(" -d ")
+                   .Append(destinationFile)
+                   .Append(" \"")
+                   .ToString();
+            processName = "/bin/bash";
+        }
+        if (processName.Length > 0) {
+            ExecuteProcess(processName, args);
+        }
+    }
+
+    public static void Create(AppCenterSettings settings)
+    {
+        var loaderZipFile = AppCenterSettingsContext.AppCenterPath + "/AppCenter/Plugins/Android/appcenter-loader-release.aar";
+        var loaderFolder = "appcenter-loader-release";
```

Comment:
"appcenter-loader-release" is duplicate a few times. Can we extract it and use string.format() instead?

Reply:
This is a temporary solution, we will revisit this one in the future.